exponent2.c

```diff
@@ -0,0 +1,26 @@
+//inefficient
```

Comment:
I like what you did with the assignment (having the user determine n!) the only thing is you forgot 2^0 = 1!